libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/Spec/Conway.hs

```diff
@@ -18,6 +18,7 @@ spec = describe "Conway conformance tests" $ do
   prop "GOV" $ conformsToImpl @"GOV" @ConwayFn @Conway
   prop "CERT" $ conformsToImpl @"CERT" @ConwayFn @Conway
   prop "RATIFY" $ conformsToImpl @"RATIFY" @ConwayFn @Conway
+  xprop "ENACT" $ conformsToImpl @"ENACT" @ConwayFn @Conway
```

Comment:
Is it not ready to be enabled yet?
```suggestion
  prop "ENACT" $ conformsToImpl @"ENACT" @ConwayFn @Conway
```

Reply:
Yeah, it's still failing, I'm just doing the translations in this PR